C-Plus-Plus/math/Euler's_Totient_function.cpp

```diff
@@ -1,49 +1,37 @@
-// Euler's Totient Function represented by M(n)
-// For integer n it calculates the no. of coprimes
-// of n from 1 to n. Do refer line 18 for 
-// better grasp of question
+// Euler's Totient Function represented by F(n)
+// denotes the no. of coprime of n from 1 to n
 #include <bits/stdc++.h>
 using namespace std;
 int F(int n){
-    if(n<=0) return 0;             // we are calculating coprime from 1 to n
-    int arr[n+1],i,j; 
-    for(i=0;i<=n;i++){
-        arr[i]=i;
-    }
+    int arr[n+1],i,j;
+    arr[0]=0,arr[1]=1;     // as 1 is coprime with itself
+    for(i=2;i<=n;i++){
+     arr[i]=i;    // initialising arr[i] with i
+    }  
     for(i=2;i<=n;i++){
-        if(arr[i]==i){             // checking if arr[i] has been untouched or not
+        if(arr[i]==i){
             for(j=i;j<=n;j+=i){
-                arr[j]-=arr[j]/i;   // here we are removing no. of coprime of j integer
-            }
+                arr[j]-=(arr[j]/i); // just like sieve of eratosthenes we are decreasing values of multiple of primes
+            }  
         }
     }
-    // till now arr[i] shows no. of coprimes of n from 0 to n
-    // Let us represent Euler's Totient Function by M(n) where n is an integer
-    // M(n) denotes the number of coprimes of n from 1 to n
-    // so till now M(n)=arr[n] i.e. we have created Euler's Totient Function Array
-    // Now we have to calculate the no. of coprime pairs between 0 and n
-    for(i=2;i<=n;i++){
-        arr[i]+=arr[i-1]; // applying dp to calculate no. of coprime 
-    }                     // between 1 and n (both 1 and n inclusive)
-    return arr[n];
+    return arr[n];  //  here each arr[i] is giving number of coprimes of i from 1 to i
 }
 int main(){
     int n;
```

Comment:
Add cout statement prompting user to enter a number. Do proper spacing to make the code more readable. (Add space between operators and operands, commas, and semicolons, and colons. Add space on the outside of parentheses and brackets)

Reply:
I have corrected the description of Euler's Totient Function in Issue #445 . Other contributors may get confused.

Reply:
@atarax665 Sir, I  have added spaces in opearators and operands. I have added comment on input statement too, so that they get a clear view of entering value.